cli.js

```diff
@@ -96,7 +103,7 @@ api.run()
 		logger.exit(api.failCount > 0 || api.rejectionCount > 0 || api.exceptionCount > 0 ? 1 : 0);
 	})
 	.catch(function (err) {
-		if (err instanceof Error) {
+		if (err.name === 'Error') {
```

Comment:
Ok, I thought so. So why do we not want to show the stack for `Error`? Why is it special-cased?